mlir/test/Dialect/Arith/invalid.mlir

```diff
@@ -215,6 +215,14 @@ func.func @func_with_ops() {
 
 // -----
 
+func.func @func_with_ops() {
+^bb0:
+  // expected-error@+1 {{op failed to verify that result type has i1 element type and same shape as operands}}
```

Comment:
error message isn't correct and this isn't a `func_with_ops`, rename and move alongside the other constant tests